src/views/download/download.jsx

```diff
@@ -233,4 +242,5 @@ var Download = React.createClass({
     }
 });
 
-render(<Page><Download /></Page>, document.getElementById('app'));
+var IntlDownload = injectIntl(Download);
```

Comment:
In other views on www, we tend to wrap `React.creatClass({})` with `injectIntl`, like this:

```javascript
var Download = injectIntl(React.createClass({
  // stuff and things
}));
```
I think it might make sense to do that here too to maintain convention, at least for now for consistency.

Reply:
+1